src/PairEvaluatorHertz.h

```diff
@@ -2,27 +2,48 @@
 // Copyright (c) 2021-2024, Auburn University
 // Part of azplugins, released under the BSD 3-Clause License.
 
-/*!
- * \file PairEvaluatorHertz.h
- * \brief Defines the pair force evaluator class for Hertz potential
- */
-
 #ifndef AZPLUGINS_PAIR_EVALUATOR_HERTZ_H_
 #define AZPLUGINS_PAIR_EVALUATOR_HERTZ_H_
 
 #include "PairEvaluator.h"
 
-#ifdef NVCC
+#ifdef __HIPCC__
 #define DEVICE __device__
 #else
 #define DEVICE
 #endif
 
+namespace hoomd
+    {
 namespace azplugins
     {
 
```

Comment:
I'll note a few minor whitespace things. Do you have pre-commit running but it didn't do anything to these?
```suggestion
```